docs/service_specification.schema.json

```diff
@@ -0,0 +1,117 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "backstage.io/v1beta1",
+    "type": "object",
+    "title": "A JSON Schema for Backstage service definitions.",
+    "description": "Each Backstage description file has a number of files. This schema matches each of those.",
+    "examples": [
+        {
+            "apiVersion": "backstage.io/v1beta1",
+            "kind": "Component",
+            "spec": {
+                "type": "service"
+            },
+            "metadata": {
+                "name": "LoremService",
+                "description": "Creates Lorems like a pro.",
+                "labels": {
+                    "product_name": "Random value Generator"
+                },
+                "annnotations": {
+                    "docs": "https://github.com/..../tree/develop/doc"
+                },
+                "teams": {
+                    "name": "Team super great",
+                    "email": "greatTeam@geemel.com"
+                }
+            }
+        }
+    ],
+    "required": [
+        "apiVersion",
+        "kind",
+        "spec",
+        "metadata"
+    ],
+    "additionalProperties": false,
+    "properties": {
+        "apiVersion": {
+            "type": "string",
+            "description": "Version of the specification format for a particular file is written against.",
+            "default": "backstage.io/v1beta1",
+            "enum": [
+                "backstage.io/v1beta1"
+            ]
+        },
+        "kind": {
+            "type": "string",
+            "description": "High level entity type being described, from the Backstage system model.",
+            "enum": [
+                "Compontent"
+            ]
+        },
+        "spec": {
+            "type": "object",
+            "description": "Actual specification data that describes the entity. TODO: shape depend on `kind`",
```

Comment:
see the main review comment - distinct kinds will probably be separate schema files